bin.js

```diff
@@ -11,6 +11,7 @@ if (a == 'init') {
   d.scripts.prepare = 'husky'
   w('package.json', JSON.stringify(d, null, /\t/.test() ? '\t' : 2) + '\n')
   process.stdout.write(i())
+	try { f.mkdirSync('.husky') } catch {}
```

Comment:
unidented commit. seems to use tab while rest of the code use spaces

Reply:
That was a mistake. It was fixed by another commit (using tabs with .editorconfig)

Reply:
Time to dogfood and setup git hooks to husky repo to auto lint? ;)

Reply:
Ahah probably :D Dog fooding json-server already.